### PR TITLE
Address AD review comments from Ben Kaduk

### PR DESCRIPTION
**Pull request by dthaler**
All but two AD comments are addressed in this PR

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

**Comment by dthaler** [comment]
@kaduk addressing most of your comments in this PR

File: draft-ietf-teep-architecture.md [hunk 1]
```diff
@@ -1327,7 +1339,11 @@ from being disclosed to the TAM that distributes them.  In such a scenario,
 the files can be encrypted end-to-end between a Trusted Component Signer and a TEE.  However, there
 must be some means of provisioning the decryption key into the TEE and/or some
 means of the Trusted Component Signer securely learning a public key of the TEE that it can use to
-encrypt.  One way to do this is for the Trusted Component Signer to run its own TAM so that it can
+encrypt.  The Trusted Component Signer cannot necessarily even trust the
```
Comment (mingpeiwk) [inline draft-ietf-teep-architecture.md]: Drop a space here: "encrypt. The..."

**Review by mingpeiwk — APPROVED** [review]
Thanks for the changes; look good.

File: draft-ietf-teep-architecture.md (hunk 1 above)
Comment (dthaler) [reply draft-ietf-teep-architecture.md]: Done

**Comment by dthaler** [comment]
Merging per Ming's approval

**Pull request merged**